src/api/app/controllers/application_controller.rb

```diff
@@ -456,4 +457,15 @@ def forward_from_backend(path)
   def shutup_rails
     Rails.cache.silence! unless Rails.env.development?
   end
+
+  def set_influxdb_tags
+    anonymous = User.current_login == User::NOBODY_LOGIN
```

Comment:
[nitpick] `User.curreny.try(:is_nobody?)` and written like that I think that the `anonymous variable is not needed

Reply:
User.current.try(:is_nobody?) is false for User.current=nil - which we don't want.